lib/spaceship/client.rb

```diff
@@ -84,14 +84,14 @@ def initialize
     end
 
     # The logger in which all requests are logged
-    # /tmp/spaceship[time].log by default
+    # /tmp/spaceship[time]_[pid].log by default
     def logger
       unless @logger
         if ENV["VERBOSE"]
           @logger = Logger.new(STDOUT)
         else
           # Log to file by default
-          path = "/tmp/spaceship#{Time.now.to_i}.log"
+          path = "/tmp/spaceship#{Time.now.to_i}_#{Process.pid}.log"
```

Comment:
I thought about using Tempfile but I didn't want to make any "drastic" changes. If you prefer to have Tempfile I can change it, without any problem.